UPGRADE-2.0.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 1.x to 2.0
+=======================
+
+### Dependencies
+
+ * The `hostnet/form-twig-bridge` depends on Symfony 3.
+ * PHP requirement has been set on >=5.6 (might work, but we did not verify)
```

Comment:
might work on versions higher than 5.6? (older versions might work, but we did not verify)